api/src/test/java/org/openmrs/ChemoDosingInstructionsTest.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License, 
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can 
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under 
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS 
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import java.util.Locale;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openmrs.test.BaseContextSensitiveTest;
+
+public class ChemoDosingInstructionsTest extends BaseContextSensitiveTest {
+
+	@Test
+	public void getDosingInstructionsAsString_shouldInferCorrectly() {
+		ChemoDosingInstructions chemoDosingInstructions = new ChemoDosingInstructions();
+		chemoDosingInstructions.setDosageAdjustmentPercentage(20.04);
+		chemoDosingInstructions.setDosageDilutionInstructions("10% diluted");
+		chemoDosingInstructions.setDosageDeliveredTiming("once in 2 hours");
+
+		String shouldBeResult = "Dosage Adjustment Percentage : 20.04\nDosage Delivered Timing : once in 2 hours"
+			+ "\nDosage Dilution Instructions : 10% diluted\n";
+
+		String dosingInstructions = chemoDosingInstructions.getDosingInstructionsAsString(Locale.getDefault());
+		Assert.assertEquals(dosingInstructions, shouldBeResult);
+	}
+
+	@Test
+	public void setDosingInstructions_shouldBeSetCorrectly() {
+		DrugOrder drugOrder = new DrugOrder();
+		ChemoDosingInstructions chemoDosingInstructions = new ChemoDosingInstructions();
+		chemoDosingInstructions.setDosageAdjustmentPercentage(20.04);
```

Comment:
Another instance of the same hard-coded strings... they are repeated throughout this class :)

Reply:
Resolved :)